### PR TITLE
Implement functionality for add/remove owners and get owned/subscribed/selected drops in backend & Convert drop_metadata objects to frontend dictionaries

### PR DESCRIPTION
**Pull request by AlexTryjan**

### CODE REVIEW

File: syncr_backend/util/drop_util.py [hunk 1]
```diff
@@ -181,6 +195,74 @@ def get_drop_metadata(
...
+
+    # Subscribed drops are those on the disk that this node does not own
+    for drop_id in drops:
+        # Get drop_metadata object for drop
+        md = simple_get_drop_metadata(drop_id)
+        if md.owner == node_id:
+            continue
+        else:
+            secondary_owner = False
+            for owner in md.other_owners:
```
[the remark this answers is not in the document]
Comment (AlexTryjan) [reply syncr_backend/util/drop_util.py]: updated

File: syncr_backend/util/drop_util.py [hunk 2]
```diff
@@ -181,6 +195,73 @@ def get_drop_metadata(
...
+    subscribed_drops = []
+
+    # Subscribed drops are those on the disk that this node does not own
+    for drop_id in drops:
+        # Get drop_metadata object for drop
+        md = simple_get_drop_metadata(drop_id)
+        if md.owner == node_id:
+            continue
+        else:
+            secondary_owner = False
```
Comment (matthewbentley) [inline syncr_backend/util/drop_util.py]: remove this

File: syncr_backend/util/drop_util.py [hunk 3]
```diff
@@ -181,6 +195,73 @@ def get_drop_metadata(
...
+            if node_id in md.other_owners:
+                continue
+            else:
```
Comment (matthewbentley) [inline syncr_backend/util/drop_util.py]: this doesn't have to be behind a else with the continue

Comment (AlexTryjan) [reply syncr_backend/util/drop_util.py]: simplified 👍

**Pull request merged**